jekyll/_cci2/nomad-metrics.md

```diff
@@ -0,0 +1,225 @@
+
+**NOTE:** Before proceeding, you should be logged into the EC2 Service section of the AWS Console.  Make sure that you logged into the region you use to run CircleCI Server.
+
+### Updating the Services Host Security Group
+
+1. Select the `Instances` link located under the Instances group in the left sidebar.
+2. Select the Services Box Instance.  The name tag typically resembles `circleci_services`.
+3. In the description box at the bottom, select the users security group link located next to the `Security Groups` section.  It typically resembles `*_users_sg`.
+4. This will take you straight to the Security Group page highlighting the users security group.  In the description box at the bottom, select the `Inbound` tab followed by the `Edit` button.
+5. Select the `Add a Rule` button.  From the drop-down, select `Custom UDP Rule`.  In the Port Range field enter `8125`.  
+6. The source field gives you a few options.  However, this ultimately depends on how you have configured the VPC and subnet.  Below are some more common scenarios.  
+   1. (Suggested) Allow traffic from the nomad client subnet.  You can usually match the entries used for ports 4647 or 3001.  For example, `10.0.0.0/24`.
+   2. Allow all traffic to UDP port 8125 using `0.0.0.0/0`.
+7. Press the `Save Button`
+
+### Updating the AWS Launch Configuration
+
+#### Prerequisites
+
+##### AWS EC2 Launch Configuration ID
+
+1. Select the `Auto Scaling Groups` (ASG) link in the the sidebar on the left.
+2. Locate the ASG with a name tag similar to`*_nomad_clients_asg`
+3. The Launch Configuration name is next to the ASG name IE `terraform-20180814231555427200000001`
+
+##### AWS EC2 Services Box Private IP Address
+
+1. Select the `Instances` link located under the Instances group in the left sidebar
+2. Select the Services Box Instance.  The name tag typically resembles `circleci_services`
+3. In the description box at the bottom of the page, make note of the private IP address.
+
+#### Updating the Launch Configuration
+
+1. Select the `Launch Configurations` link located under `Auto Scaling` in the sidebar to the left.  Select the Launch Configuration you retrieved in the previous steps.
+2. In the description pane at the bottom, select the `Copy launch configuration` button.
+3.  Once the configuration page opens, select `3. Configure details` link located at the top of the page.
+4. Update the `Name` field to something meaningful IE `nomad-client-with-metrics-lc-DATE`.
+5. Select the `Advanced Details` drop down.
+6. Copy and paste the launch configuration script from below in the text field next to `User data`.
+7. **IMPORTANT:** Enter the private IP address of the services box at Line 10. For example, `export SERVICES_PRIVATE_IP="192.168.1.2"`.
+8. Select the `Skip to review` button and then`Create launch configuration` button.
```

Comment:
I was also asked about selecting an existing key / pair. Should this be added. 
![image](https://user-images.githubusercontent.com/435563/44370865-fe178400-a490-11e8-9a99-3ab8c81b32be.png)